Common subdirectories: xinit-1.3.4/launchd and xinit-1.3.4.new/launchd
Common subdirectories: xinit-1.3.4/man and xinit-1.3.4.new/man
diff -up xinit-1.3.4/xinitrc.cpp xinit-1.3.4.new/xinitrc.cpp
--- xinit-1.3.4/xinitrc.cpp	2014-03-27 21:37:38.000000000 +0800
+++ xinit-1.3.4.new/xinitrc.cpp	2015-07-19 15:35:34.375850468 +0800
@@ -2,21 +2,13 @@ XCOMM!SHELL_CMD
 
 userresources=$HOME/.Xresources
 usermodmap=$HOME/.Xmodmap
-sysresources=XINITDIR/.Xresources
-sysmodmap=XINITDIR/.Xmodmap
+sysresources=XINITDIR/Xresources
+sysmodmap=XINITDIR/Xmodmap
 
 XCOMM merge in defaults and keymaps
 
 if [ -f $sysresources ]; then
-#ifdef __APPLE__
-    if [ -x /usr/bin/cpp ] ; then
-        XRDB -merge $sysresources
-    else
-        XRDB -nocpp -merge $sysresources
-    fi
-#else
     XRDB -merge $sysresources
-#endif
 fi
 
 if [ -f $sysmodmap ]; then
@@ -24,15 +16,7 @@ if [ -f $sysmodmap ]; then
 fi
 
 if [ -f "$userresources" ]; then
-#ifdef __APPLE__
-    if [ -x /usr/bin/cpp ] ; then
-        XRDB -merge "$userresources"
-    else
-        XRDB -nocpp -merge "$userresources"
-    fi
-#else
     XRDB -merge "$userresources"
-#endif
 fi
 
 if [ -f "$usermodmap" ]; then
@@ -42,14 +26,8 @@ fi
 XCOMM start some nice programs
 
 if [ -d XINITDIR/xinitrc.d ] ; then
-	for f in XINITDIR/xinitrc.d/?*.sh ; do
+	for f in XINITDIR/xinitrc.d/?* ; do
 		[ -x "$f" ] && . "$f"
 	done
 	unset f
 fi
-
-TWM &
-XCLOCK -geometry 50x50-1+1 &
-XTERM -geometry 80x50+494+51 &
-XTERM -geometry 80x20+494-0 &
-exec XTERM -geometry 80x66+0+0 -name login
